addon/styles/components/form-field.scss

```diff
@@ -1,3 +1,8 @@
 .form-field {
   display: inline;
+
+  div {
+    width: 70%;
+    display: inline-block;
+  }
```

Comment:
Why did you add this? I think it doesn't belong here. It's something you just can add in de resource-manager, isn't it?